src/euphorie/client/browser/risk.py

```diff
@@ -348,7 +352,12 @@ def _prepare_risk(self):
 
         # Italian special
         if self.webhelpers.country == "it":
-            self.skip_evaluation = True
+            if self.risk and (
+                self.risk.type == "top5" or self.risk.evaluation_method == "fixed"
+            ):
+                self.skip_evaluation = True
+            else:
+                self.evaluation_condition = "condition: answer=no or answer=yes"
```

Comment:
Can `skip_evaluation` and `evaluation_condition` those be properties?
Setting class attributes on the fly is hard to debug and also this could shave some lines from the very very long `_prepare_risk` method.

Reply:
ok, done